guides/client-json-configuration-example.md

```diff
@@ -0,0 +1,181 @@
+# Client JSON Configuration Example
+
+This document provides an example JSON configuration that can be used with `Client.fromConfig()` across the Hiero SDKs. The configuration defines all necessary parameters to connect to and interact with a Hedera network.
+
+## Example Configuration
+
+```json
+{
+  "network": {
+    "35.242.233.154:50211": "3.5.10",
+    "35.240.118.96:50211": "3.5.11",
+    "34.127.45.23:50211": "3.5.12"
+  },
+  "mirrorNetwork": [
+    "hcs.testnet.mirrornode.hedera.com:443"
+  ],
+  "shard": 0,
+  "realm": 0,
+  "grpcDeadline": 10000,
+  "requestTimeout": 120000
+}
+```
+
+## Configuration Parameters
+
+### Network Configuration
+
+- **`network`** (object, required): Maps consensus node addresses to their corresponding version numbers
+  - **Key format**: `"IP:PORT"` (e.g., `"35.242.233.154:50211"`)
+  - **Value format**: `"MAJOR.MINOR.PATCH"` version string (e.g., `"3.5.10"`)
```

Comment:
This should be the value of the account ID for that node (so `3.5.10` would be the associated account in this example)